Core/Core/UITestHelpers/UITestHelpers.swift

```diff
@@ -148,7 +148,7 @@ public class UITestHelpers {
         case .useMocksOnly:
             MockDistantURLSession.setup()
         case .debug:
-            // insert ad-hoc debug code here
+            // insert ad-hoc debug code her
```

Comment:
was this intentional? 😆

Reply:
Nope, XVim sometimes likes to eat the last character when I paste something 🤦‍♂